src/containers/consumerOnboarding/components/sandbox/SandboxAccessForm.tsx

```diff
@@ -169,13 +183,22 @@ const SandboxAccessForm: FC<SandboxAccessFormProps> = ({ onSuccess }) => {
             );
           }}
         </Formik>
-        {submissionError && (
+        {submissionHasError && (
           <AlertBox
             status="error"
             headline="We encountered a server error while saving your form. Please try again later."
             content={
               <span>
                 Need assistance? Create an issue through our <Link to="/support">Support page</Link>
+                {submissionErrors.length > 0 ? (
+                  <ul>
+                    {submissionErrors.map((item: string) => (
+                      <li key={item}>{item}</li>
+                    ))}
+                  </ul>
+                ) : (
+                  ''
+                )}
```

Comment:
would generally prefer to avoid a JSX ternary if the falsy expression isn't rendering anything

```suggestion
                {submissionErrors.length > 0 && (
                  <ul>
                    {submissionErrors.map((item: string) => (
                      <li key={item}>{item}</li>
                    ))}
                  </ul>
                )}
```

Reply:
I agree.